frontend/src/pages/archive/crawl-templates-list.ts

```diff
@@ -177,108 +179,105 @@ export class CrawlTemplatesList extends LiteElement {
         </div>
       </div>
 
-      ${this.crawlTemplates && this.crawlTemplates.length
```

Comment:
Mostly an indentation change after removing `      ${this.crawlTemplates && this.crawlTemplates.length`